uktrade/great-cms
# KLS-781 -  Export academy new user sign up

This PR adds the journey for new users to great to sign up via the export academy.

It builds on the existing sign up implementation, using a query parameter `?existing-ea-user=true` to identify whether the user is new or already exists in the export academy. This then gets passed through to the html template via context and is used to determine which content to show.

**Screenshots**
<img width="1530" alt="Screenshot 2023-06-26 at 11 19 35" src="https://github.com/uktrade/great-cms/assets/22460823/963e0c5d-1961-4fec-91a6-d11ca5239dd2">

<img width="1512" alt="Screenshot 2023-06-26 at 11 20 46" src="https://github.com/uktrade/great-cms/assets/22460823/89f85738-9748-470b-ba9c-5cc5e2d37d43">

<img width="1503" alt="Screenshot 2023-06-26 at 11 21 10" src="https://github.com/uktrade/great-cms/assets/22460823/2e68f99e-a316-4137-83db-083bfb49562b">

**To test**
- Go to http://greatcms.trade.great:8020/export-academy/signup
- Make sure you've got forms API, sso and sso-proxy running
- Sign up with an email address and continue through the flow
- You should eventually be redirected to the export academy events list page after seeing sign up complete

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-781

### Reviewing help


### Housekeeping


### Merging
